docs/releases/2.2.txt

```diff
@@ -265,6 +265,11 @@ Tests
 * The new :setting:`ORACLE_MANAGED_FILES <TEST_ORACLE_MANAGED_FILES>` test
   database setting allows using Oracle Managed Files (OMF) tablespaces.
 
+* Deferrable database constraints are now checked at the end of each
+  :class:`~django.test.TestCase` test on SQLite 3.20+ as well. These checks
```

Comment:
I think you could drop "as well" (or at least clarify what "as well" refers to). I believe the meaning is that these checks are already performed on other databases.

In fcd08c175787e909b3eb98f756317a07741c48dd, this was also mentioned as a backwards incompatible change.